Dockerfile

```diff
@@ -394,6 +394,19 @@ ENV GIMME_GO_VERSION "1.10"
 ENV GIMME_ENV_PREFIX "/opt/buildhome/.gimme/env"
 RUN gimme
 
+################################################################################
+#
+# Erlang+Elixir
+#
+################################################################################
+USER root
+# Erlang solutions repository
+RUN wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && \
+    dpkg -i erlang-solutions_1.0_all.deb
+RUN apt-get update
```

Comment:
Is it possible to add this repository using `add-apt-repository`? If so, it would be better to include these installs in the giant `apt` command at the start of the `Dockerfile`. That way it only has to run `apt-get update` once.

Reply:
It looks like the official erlang instructions under "Alternatively: adding the repository entry manually" apply. I think this would do it:

```
apt-key adv --fetch-keys https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc && \
apt-add-repository -y 'deb https://packages.erlang-solutions.com/ubuntu trusty contrib' && \
```
and lower down in the list of packages add `esl-erlang` and `elixir`.

Reply:
Cool! I will try this and update the PR once I got something working.